charts/incubator/lldap/questions.yaml

```diff
@@ -4,25 +4,19 @@ portals:
 # Include{portalLink}
 questions:
 # Include{global}
-# Include{controllerExpert}
-# Include{fixedEnv}
-# Include{containerConfig}
+# Include{workload}
+# Include{workloadDeployment}
+# Include{replicas1}
+# Include{podSpec}
+# Include{containerMain}
   - variable: env
+    label: ""
     group: "App Configuration"
-    label: "Image Environment"
     schema:
       additional_attrs: true
       type: dict
       attrs:
-        - variable: LLDAP_LDAP_USER_PASS
```

Comment:
Removed this one as it's easy enough to change in app I don't think specifying this adds value.